fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/RepositoryServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.impl.services;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.fcrepo.kernel.api.Transaction;
+import org.fcrepo.kernel.api.services.RepositoryService;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.util.Collection;
+
+/**
+ * Implementation of {@link org.fcrepo.kernel.api.services.RepositoryService}
+ *
+ * @author dbernstein
+ */
+@Component
+public class RepositoryServiceImpl extends AbstractService implements RepositoryService {
```

Comment:
This is a hold over service, but I can see value to the getRepositorySize and getRepositoryObjectCount functions. The backup and restore will be removed I think.

Reply:
I agree.  I can remove backup and restore.